invenio/templates/deployments/web.yaml

```diff
@@ -181,7 +181,7 @@ spec:
         {{- end }}
       {{- if .Values.logstash.enabled }}
       - name: filebeat
-        image: {{ .Values.logstash.filebeat_image }}
+        image: elastic/filebeat:7.17.13
```

Comment:
`filebeat-oss`
why not keeping the variable as before? We should, so we can easily change the version via config.

Reply:
ok, changed